### ISSUE TITLE
Introduce "Scaler Contributor" Credly badge

### ISSUE THREAD

**Issue by tomkerkhove** [kedacore/governance#58]
Introduce "Scaler Contributor" Credly badge for which scaler contributors can apply through an issue on this repo when they have met the requirements.

Relates to https://github.com/kedacore/governance/issues/30

## Tasklist
- [x] Create an issue template for people to apply for the badge
- [x] Create a Credly badge
- [x] Define the requirements
- [x] Reach out to all previous scaler contributors that meet the criteria
- [x] Document badges and criteria

**Comment by tomkerkhove** [comment]
Still working on this though, just behind the scenes.

**Comment by tomkerkhove** [comment]
Bumped again in CNCF Service desk

**Comment by tomkerkhove** [comment]
Escalated ticket on CNCF Service Desk. /cc @caniszczyk

**Comment by tomkerkhove** [comment]
Everything was sent to CNCF team for following badges:
- Maintainer
- Scaler Maintainer (not Contributor)
- Contributor

Designs are being added as part of https://github.com/kedacore/governance/pull/112

**Comment by tomkerkhove** [comment]
Badges are issued, all I need to do is document things here

**Comment by tomkerkhove** [comment]
Doc PR: https://github.com/kedacore/governance/pull/113

**Comment by JorTurFer** [comment]
Maybe we have to include some suggestions about how to earn these badges, things like:
- Amount of contributions
- Type of contributions
- Required period
- Nomination process (will we do it or they can self-nominate?)
- Etc..

**Comment by zroubalik** [comment]
Yeah, I agree, maybe we don’t have to specify the exact data, but we can mention that in order to be eligible, one needs to work consistently and not just make a one-time contribution.

**Comment by rickbrouwer** [comment]
The link 'Earn this badge' on the [contributor badge on Credly](https://www.credly.com/org/the-linux-foundation/badge/keda-contributor) currently points to the following page:

https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md

However, there is no information about earning the badge there.
I think the badge (and the other KEDA badges too) would be better to point to:

https://github.com/kedacore/governance/blob/main/COMMUNITY.md

I think it is good to also set the criteria so that the maintainers may get a list so that you can show the appreciation to essential members of our community. Personally, I think that's better than explaining why you believe you qualify for a badge.
